IssueCreator/IssueCreator/IssueManager.cs

```diff
@@ -0,0 +1,270 @@
+        {
+            try
+            {
+                await _zenHubClient.GetEpicClient(repoId, epicNumber).AddIssuesAsync(new[] { Helpers.ObjectCreator.CreateIssue(issueNumber, repoIdIssue) });
+                return true;
+            }
+            catch
+            {
+                return false;
+            }
+        }
+
+        public async Task SetIssueEstimateAsync(long repoId, int issueNumber, int estimate)
+        {
+            if (estimate > 0)
+            {
+                await _zenHubClient.GetIssueClient(repoId, issueNumber).SetEstimateAsync(estimate);
+            }
+        }
+
+        public async Task ConvertToEpicAsync(long repoId, int issueNumber)
+        {
+            await _zenHubClient.GetIssueClient(repoId, issueNumber).ConvertToEpicAsync(Enumerable.Empty<Issue>());
+        }
+
+        public async Task<Issue> CreateIssueAsync(string owner, string repo, string title, string description, string assignedTo, List<string> labels, int? milestone = null)
+        {
+            NewIssue issue = new NewIssue(title);
+
+            foreach (var item in labels)
+            {
+                issue.Labels.Add(item.ToString());
+            }
+
+            issue.Body = description;
+            issue.Milestone = milestone;
+
+            if (!string.IsNullOrEmpty(assignedTo))
+            {
+                issue.Assignees.Add(assignedTo);
+            }
+
+            return await _githubClient.Issue.Create(owner, repo, issue);
+        }
+
+        public async Task<Repository> GetRepositoryAsync(string owner, string repository)
+        {
+            return await GetValueFromCache($"repo_{owner}_{repository}", () => _githubClient.Repository.Get(owner, repository));
+        }
+
+        internal async Task<(bool, string)> TryCreateNewIssueAsync(IssueToCreate issueToCreate)
+        {
+            bool validEstimate = false;
+            int estimate = 0;
+            if (string.IsNullOrEmpty(issueToCreate.Estimate))
+            {
+                validEstimate = true;
+            }
+            else if (int.TryParse(issueToCreate.Estimate, out estimate) && estimate > 0)
+            {
+                validEstimate = true;
+            }
+
+            if (!validEstimate)
+            {
+                return (false, "Estimate needs to be a positive number greater than zero.");
+            }
+
+            // Create the issue on GitHub
+            var createdIssue = await CreateIssueAsync(
+                issueToCreate.Organization,
+                issueToCreate.Repository,
+                issueToCreate.Title,
+                issueToCreate.Description,
+                issueToCreate.AssignedTo?.ToString(),
+                issueToCreate.Labels,
+                issueToCreate.Milestone.Number);
+
+            var repoFromGH = await GetRepositoryAsync(issueToCreate.Organization, issueToCreate.Repository);
```

Comment:
Nit - maybe wrap the remainder in a try/catch to alert the user they should go set the epic by hand if we fail after creating the github issue?